eng/versioning/version_client.txt

```diff
@@ -2,7 +2,7 @@
 # <groupId>:<artifactId>;dependency-version;current-version
 
 com.azure:azure-sdk-all;1.0.0;1.0.0
-com.azure:azure-sdk-parent;1.6.0;1.6.0
+com.azure:azure-sdk-parent;1.7.0;1.7.0
```

Comment:
@JonathanGiles Does removal of the maven start jetty task from the parent/pom.xml actually require a pom version increase? It's unclear, to me, if it does however, if we do increase the version then I'm pretty sure we're going to need to publish updated versions during the next release.

Reply:
I'll do some validation around this before I merge this because I'd like to avoid it if we can. But I think because that POM is not being used as an aggregate and is not strictly listed as a dependency then Maven will try to pull the version from Maven Central rather than using the local copy.

But I need to test it in isolation because when I experienced the problems that made me bump this version there were lots of other issues going on.

Reply:
Looks like we got away with not having to bump the versions!

Reply:
Thank you for confirming @mitchdenny! I'm very cautious about increasing the parent pom versions unless absolutely necessary.